<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VirtualConcierge</class>
 <widget class="QWidget" name="VirtualConcierge">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>966</width>
    <height>731</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Virtual Concierge</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="7">
    <widget class="QLabel" name="label_name">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>100</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Arial Narrow</family>
       <pointsize>48</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Faculty of Engineering</string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="button_wheelchair">
     <property name="minimumSize">
      <size>
       <width>128</width>
       <height>32</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>128</width>
       <height>32</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">/* Button Styles*/
QPushButton {
  background: rgba(45, 46, 48, 50%);
  color: white;
  font: bold;
  border-radius: 4px;
}
QPushButton:hover:!pressed { 
  border: 0px solid  rgba(0, 115, 199, 50%);
  background: rgba(0, 115, 199, 50%);
  border-radius: 4px;
}
QPushButton:checked { 
  border: 0px solid  rgba(0, 115, 199, 50%);
  background:  rgba(0, 115, 199, 50%);
    border-radius: 4px;
}</string>
     </property>
     <property name="text">
      <string>Wheelchair</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QPushButton" name="button_feet">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>32</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>32</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">/* Button Styles*/
QPushButton {
  background: rgba(45, 46, 48, 50%);
  color: white;
  font: bold;
  border-radius: 4px;
}
QPushButton:hover:!pressed { 
  border: 0px solid  rgba(0, 115, 199, 50%);
  background: rgba(0, 115, 199, 50%);
  border-radius: 4px;
}
QPushButton:checked { 
  border: 0px solid  rgba(0, 115, 199, 50%);
  background:  rgba(0, 115, 199, 50%);
    border-radius: 4px;
}</string>
     </property>
     <property name="text">
      <string>Feet</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="4">
    <widget class="QPushButton" name="button_bicycle">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>32</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>32</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">/* Button Styles*/
QPushButton {
  background: rgba(45, 46, 48, 50%);
  color: white;
  font: bold;
  border-radius: 4px;
}
QPushButton:hover:!pressed { 
  border: 0px solid  rgba(0, 115, 199, 50%);
  background: rgba(0, 115, 199, 50%);
  border-radius: 4px;
}
QPushButton:checked { 
  border: 0px solid  rgba(0, 115, 199, 50%);
  background:  rgba(0, 115, 199, 50%);
    border-radius: 4px;
}</string>
     </property>
     <property name="text">
      <string>Bicycle</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="5">
    <widget class="QPushButton" name="button_other_vehicle">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>128</width>
       <height>32</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>128</width>
       <height>32</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">/* Button Styles*/
QPushButton {
  background: rgba(45, 46, 48, 50%);
  color: white;
  font: bold;
  border-radius: 4px;
}
QPushButton:hover:!pressed { 
  border: 0px solid  rgba(0, 115, 199, 50%);
  background: rgba(0, 115, 199, 50%);
  border-radius: 4px;
}
QPushButton:checked { 
  border: 0px solid  rgba(0, 115, 199, 50%);
  background:  rgba(0, 115, 199, 50%);
    border-radius: 4px;
}</string>
     </property>
     <property name="text">
      <string>Other Vehicle</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="7">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QWidget" name="widget_directory" native="true">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>256</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">/* Button Styles*/
QPushButton {
  background: rgba(45, 46, 48, 25%);
  color: white;
  font: bold;
  border-radius: 4px;
}
QPushButton:hover:!pressed { 
  border: 0px solid  rgba(0, 115, 199, 25%);
  background: rgba(0, 115, 199, 25%);
  border-radius: 4px;
}
QPushButton:checked { 
  border: 0px solid  rgba(0, 115, 199, 25%);
  background:  rgba(0, 115, 199, 25%);
    border-radius: 4px;
}</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="VirtualConciergeRenderstate" name="openGLWidget"/>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="pushButton_send_mail">
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>32</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">/* Button Styles*/
QPushButton {
  background: rgba(45, 46, 48, 50%);
  color: white;
  font: bold;
  border-radius: 4px;
}
QPushButton:hover:!pressed { 
  border: 0px solid  rgba(0, 115, 199, 50%);
  background: rgba(0, 115, 199, 50%);
  border-radius: 4px;
}
QPushButton:checked { 
  border: 0px solid  rgba(0, 115, 199, 50%);
  background:  rgba(0, 115, 199, 50%);
    border-radius: 4px;
}</string>
     </property>
     <property name="text">
      <string>Mail</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="lineEdit_email">
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>32</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
  background: rgba(45, 46, 48, 50%);
  color: white;
  font: bold;
  border-radius: 4px;
}

QLineEdit:hover {
  border: 0px solid  rgba(0, 115, 199, 50%);
  background: rgba(0, 115, 199, 50%);
  border-radius: 4px;
}

QLineEdit:pressed
{
  border: 0px solid  rgba(0, 115, 199, 50%);
  background: rgba(0, 115, 199, 50%);
  border-radius: 4px;
}
</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4" colspan="2">
    <widget class="QLabel" name="label_2">
     <property name="maximumSize">
      <size>
       <width>1024</width>
       <height>50</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Monotype Corsiva</family>
       <pointsize>24</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>Virtual Concierge </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="button_play_pause">
     <property name="minimumSize">
      <size>
       <width>256</width>
       <height>32</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">/* Button Styles*/
QPushButton {
  background: rgba(45, 46, 48, 25%);
  color: white;
  font: bold;
  border-radius: 4px;
}
QPushButton:hover:!pressed { 
  border: 0px solid  rgba(0, 115, 199, 25%);
  background: rgba(0, 115, 199, 25%);
  border-radius: 4px;
}
QPushButton:checked { 
  border: 0px solid  rgba(0, 115, 199, 25%);
  background:  rgba(0, 115, 199, 25%);
    border-radius: 4px;
}</string>
     </property>
     <property name="text">
      <string>Playing</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>VirtualConciergeRenderstate</class>
   <extends>QOpenGLWidget</extends>
   <header>RenderStates/virtualconciergerenderstate.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
